assets/js/modules/analytics/settings/settings-form.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Analytics Settings form.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	AccountSelect,
+	AnonymizeIPSwitch,
+	ExistingTagNotice,
+	ProfileSelect,
+	PropertySelect,
+	TrackingExclusionSwitches,
+	UseSnippetSwitch,
+} from '../common/';
+
+export default function SettingsForm() {
+	return (
+		<div className="googlesitekit-analytics-settings-fields">
+			<ExistingTagNotice />
+
+			<div className="googlesitekit-setup-module__inputs">
+				<AccountSelect />
+
+				<PropertySelect />
+
+				<ProfileSelect />
+			</div>
+
+			<div className="googlesitekit-setup-module__inputs googlesitekit-setup-module__inputs--multiline">
+				<UseSnippetSwitch />
+
+				<AnonymizeIPSwitch />
+
+				<p className="googlesitekit-setup-module__text">
```

Comment:
Should this be in the `TrackingExclusionSwitches` component instead? It's directly-related to them and I don't see why you wouldn't want this context in the component itself.

If you did, you could disable it with a prop. But this `<p>` seems like it should be in `<TrackingExclusionSwitches>`.

Reply:
Agreed. Also technically for accessibility this should probably be a `fieldset` `legend` - doing that here would probably mess up the styling though, so if that's the case, we can keep the `p`. Accessibility is something that needs to be looked at holistically at some point - we have TONS of issues.

Reply:
It is an outlier compared to the others so I'll see what we can do about that here. I think the reason I left it out was because the others have the label built into the underlying component. This one is actually a list of `Switch` components.

That reminds me - @felixarntz - Matt and I discussed this a bit before but what would you think about making selectors for the individual values of `trackingDisabled` rather than dealing with the array value directly since this is more of an implementation detail? In this case, I think it would make more sense to parse the array with selectors than deal with the whole thing externally (unless we really need to). This would also simplify logic around `undefined` handling I think.

Reply:
Using a `fieldset` and `legend` looks fine to me 👍 

![image](https://user-images.githubusercontent.com/1621608/79573357-34266480-80c7-11ea-9aa8-bf2d1bd447fb.png)

Updated ✅